x-pack/test/apm_api_integration/tests/traces/top_traces.spec.ts

```diff
@@ -78,6 +79,7 @@ export default function ApiTest({ getService }: FtrProviderContext) {
 
         expectSnapshot(lastItem).toMatchInline(`
           Object {
+            "agentName": "dotnet",
```

Comment:
It would be great if you can change this test to use synthtrace and the new [`apmApiClient` instead of the legacy `supertest` service](https://github.com/elastic/kibana/issues/111308).

Reply:
@sqren good point here. I would rather do it in another PR though in order to move this one forward